prov/gni/include/gnix_vector.h

```diff
@@ -86,7 +87,7 @@ struct gnix_vector;
 
 struct gnix_vector_iter {
 	struct gnix_vector *vec;
-	int cur_idx;
+	uint64_t cur_idx;
```

Comment:
No need to modify it. We'll never make a vector that large. It would take too much memory away from the customer applications to be a reasonable use case.